### ISSUE TITLE
Phone: Vertical burger icon list is broken

### ISSUE THREAD

**Issue by gerroon** [fatg3erman/RompR#35]
Tablet-Darkness-ModernLight-Nunito, the vertical burge icon drop downs nothing. I only see a smal black box when I press on it. I am using Chromium (also tested waterfox) on Android.

The actual "Phone" skin works. It seems to be a Tablet skin thing on 1.16 THis was fine prior.

**Comment by fatg3erman** [comment]
Works for me. This will probably be one of the annoying browser caching issues that happen, despite my best efforts, especially with Chrome.

Try refreshing the browser a few times. If that doesn't work you'll have to clear your cache and reload the page. If you're using a home screen icon on a phone or tablet you'll need to delete it and recreate it. 

Other browsers usually behave correctly, but Chrome/Chromium sometims refuses to refresh its cache even if I explicitly tell it to.

**Comment by gerroon** [comment]
Yeah I remember that from the docs. That is why I tried it with FIrefox which it had no cache with the same result, but I will try clearing Chromium cache.

**Comment by fatg3erman** [comment]
OK found it. It's screen size dependant, hence why I didn't see it at first. I can reproduce it in the phone emulator in Chrome. Bug.
